go/libkb/leveldb.go

```diff
@@ -367,6 +372,33 @@ func (l *LevelDb) OpenTransaction() (LocalDbTransaction, error) {
 	return ltr, nil
 }
 
+func (l *LevelDb) KeysWithPrefixes(prefixes ...[]byte) (DBKeySet, error) {
```

Comment:
I'm not sure why this is needed....  I wouldn't have guessed it was needed... Is this going to be fast enough?

Reply:
I needed some way to select a random team to audit, thought it would be easiest to just use the teamload/ftl cache. I believe finding all keys with a certain prefix is fast because of leveldb's architecture where they are sorted